src/compile.ts

```diff
@@ -203,16 +203,16 @@ export default async function ({
    */
   const program = typescript.createProgram(productionSourceFiles, {
     module: typescript.ModuleKind.ESNext,
-    moduleResolution: typescript.ModuleResolutionKind.Bundler,
     target: typescript.ScriptTarget.ESNext,
+    moduleResolution: typescript.ModuleResolutionKind.Bundler,
```

Comment:
i changed the order to exactly match `tsconfig.json`, so easier to do side-by-side comparison